practicals/GNN_for_recommendations.ipynb

```diff
@@ -0,0 +1,1284 @@
+{
```

Comment:
<p>Repeated title (repeat of the one just above).</p><p> </p><p>I'm guessing you meant for this cell to explain the train loop that appears just below?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/deep-learning-indaba/indaba-pracs-2024/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='44'/>